ZMQWindows.md

```diff
@@ -10,25 +12,32 @@
+> As of Nov 2019, ZMQ 4.3.3 does not work and results in an error: https://github.com/pebbe/zmq4/issues/152/. 4.3.2 is recommended and tested on Win 10:
+https://ci.appveyor.com/project/zeromq/libzmq/builds/25882500/job/s2yhbnjrdouq5qnm/artifacts
+and
+https://github.com/zeromq/libzmq/tree/a84ffa12b2eb3569ced199660bac5ad128bff1f0. 
 
-4. Copy over the `include` directory from the source downloaded in step 3 to  `%GOPATH%\src\github.com\pebbe\zmq4\usr\local\include`, ensuring that any necessary parent directories that don't exist are created.
+5. Copy over the `include` directory from the source downloaded in step 3 to  `%GOPATH%\pkg\mod\github.com\pebbe\zmq4@v1.0.0\`.
 
-5. Set the following environment variables (run this in PowerShell):
+6. Set the following environment variables (run this in PowerShell):
 
 ```
-$Env:CGO_CFLAGS="-I$Env:GOPATH\src\github.com\pebbe\zmq4\usr\local\include"
-$Env:CGO_LDFLAGS="-L$Env:GOPATH\src\github.com\pebbe\zmq4\usr\local\lib"
+$Env:CGO_CFLAGS="-I$Env:GOPATH\pkg\mod\github.com\pebbe\zmq4@v1.0.0\include"
+$Env:CGO_LDFLAGS="-L$Env:GOPATH\pkg\mod\github.com\pebbe\zmq4@v1.0.0\lib"
 ```
 
 After setting those, running `go get -v -x github.com/pebbe/zmq4` should succeed. If it fails, it most likely due to the `.h` files or `dll`s existing in a directory that doesnt match your `CGO_` environment variables. It may also fail due to the same error as before (see step 3.b) - try downloading the artifact from a different build on the [AppVeyor builds site](https://ci.appveyor.com/project/zeromq/libzmq).
 
-5. Next, navigate to `$Env:GOPATH\src\github.com\edgexfoundry\edgex-go\cmd\core-data`. *If you didn't acquire dependencies previously, run `go get -v` to acquire all dependencies*. Run `go build -v`. If it complains about not finding zmq.h from auth.go, double check the CGO_ environment variables by running `go env`.
+> It is recommended to set these environment variables in your System or IDE so that you do not need to set them in every terminal session. This especially required if you wish to debug services that depend on ZMQ. 
+
+7. Next, navigate to `$Env:GOPATH\src\github.com\edgexfoundry\edgex-go\cmd\core-data`. *If you didn't acquire dependencies previously, run `go get -v` to acquire all dependencies*. Run `go build -v`. If it complains about not finding zmq.h from auth.go, double check the CGO_ environment variables by running `go env`.
 
-6. There should now be a `core-data.exe` file in your directory. The last step is to ensure that a copy of the DLL's `libsodium.dll` and `libzmq-v120-mt-4_x_x.dll` (yes, this one, not with the rename we did earlier) is in the same directory as your `.exe`.
+8. There should now be a `core-data.exe` file in your directory. The last step is to ensure that a copy of the DLL's `libsodium.dll` and `libzmq-v120-mt-4_x_x.dll` (yes, this one, not with the rename we did earlier) is in the same directory as your `.exe`.

```

Comment:
This is more of something I've run into while following this setup guide. After completing Step 7, I am not seeing the libsodium.dll and libzmq-v120-mt4_x_x.dll in the same directory as my core-data.exe.

With that, when I execute core-data.exe, I receive the following output:
`
$ ./core-data.exe
C:/Users/Jason/go/src/github.com/edgexfoundry/edgex-go/cmd/core-data/core-data.exe: error while loading shared libraries: libzmq-v141-mt-4_3_2.dll: cannot open shared object file: No such file or directory
`
I would assume that you are not seeing this. Any possible pointers as to why this may be?

Reply:
Just wanted to add that, when I manually copy `libsodium.dll` and `libzmq-v141-mt-4_x_x.dll` to the directory in which my `exe` is, the application runs as expected. Not sure if that is a manual step or something that should happen following the steps in the guide.

Reply:
You are correct that this is a manual step, it will not be done for you. I will reword it to make it more clear. Thanks